### ISSUE TITLE
Auras issues

### ISSUE THREAD

**Issue by YokiHotsumi** [Niitek/LUI-WotLK#22]
Hello,

I played rogue, and i have too many errors with weapons poisons auras.

> 21x bad argument #2 to '?' (Usage: self:SetVertexColor(color [, a]))
[string "=[C]"]: in function `SetVertexColor'
[string "@LUI/modules/auras/auras.lua"]:251: in function `Update'
[string "@LUI/modules/auras/auras.lua"]:385: in function `UpdateWeaponEnchants'
[string "@LUI/modules/auras/auras.lua"]:346: in function <LUI/modules/auras/auras.lua:335>
[string "=[C]"]: ?
Locals:
(*temporary) = "bad argument #2 to '?' (Usage: self:SetVertexColor(color [, a]))"

**Comment by Ayiadim** [comment]
I've got a fix for this, will be in the next release.

**Comment by YokiHotsumi** [comment]
With LUI-3.50.02, i have the same issue. 

```
5x bad argument #2 to '?' (Usage: self:SetVertexColor(color [, a]))
[string "=[C]"]: in function `SetVertexColor'
[string "@LUI/modules/auras/auras.lua"]:251: in function `Update'
[string "@LUI/modules/auras/auras.lua"]:385: in function `UpdateWeaponEnchants'
[string "@LUI/modules/auras/auras.lua"]:346: in function <LUI/modules/auras/auras.lua:335>
[string "=[C]"]: ?

Locals:
(*temporary) = "bad argument #2 to '?' (Usage: self:SetVertexColor(color [, a]))"
```

**Comment by YokiHotsumi** [comment]
and this one too

```
1x LUI/modules/auras/auras.lua:250: Usage: GetItemQualityColor(index)
[string "=[C]"]: ?
[string "@LUI/modules/auras/auras.lua"]:250: in function `Update'
[string "@LUI/modules/auras/auras.lua"]:385: in function `UpdateWeaponEnchants'
[string "@LUI/modules/auras/auras.lua"]:346: in function <LUI/modules/auras/auras.lua:335>
[string "=[C]"]: ?

Locals:
(*temporary) = nil
```

**Comment by Niitek** [comment]
Issue has been resolved and working correctly in v3.50.03
